code/game/objects/structures/crates_lockers/closets/secure/scientist.dm

```diff
@@ -47,3 +47,4 @@
 	new /obj/item/clothing/mask/gas(src)
 	new /obj/item/device/flash(src)
 	new /obj/item/weapon/switchtool/holo(src)
+	new /obj/item/clothing/shoes/kneesocks/pinkknee(src)
```

Comment:
Right, the image has to have a short and descriptive name (It's pink and has something to do with knees) but right here you've said it's something to do with knees twice.

shoes don't do this, shoes are just shoes/black, shoes/orange, shoes/brown